Yafc/Widgets/ObjectTooltip.cs

```diff
@@ -193,6 +193,27 @@ private void BuildCommon(FactorioObject target, ImGui gui) {
             }
             else {
                 gui.BuildText(CostAnalysis.GetDisplayCost(target), TextBlockDisplayStyle.WrappedText);
+
+                // Show cost breakdown if Control is held
+                if (InputSystem.Instance.control) {
+                    string breakdown = CostAnalysis.GetCostBreakdown(target, true); // Show current milestone costs
+                    if (!string.IsNullOrEmpty(breakdown)) {
+                        gui.BuildText("", TextBlockDisplayStyle.WrappedText); // Add some spacing
+                        gui.BuildText("Cost Breakdown (Current Milestones):", TextBlockDisplayStyle.Default(SchemeColor.BackgroundText));
+
+                        // Split the breakdown into lines and display each separately
+                        string[] lines = breakdown.Split('\n');
+                        foreach (string line in lines) {
+                            if (!string.IsNullOrWhiteSpace(line)) {
+                                gui.BuildText(line, TextBlockDisplayStyle.Default(SchemeColor.BackgroundTextFaint));
+                            }
+                        }
```

Comment:
```suggestion
                        gui.BuildText(breakdown, TextBlockDisplayStyle.WrappedText with { Color = SchemeColor.BackgroundTextFaint });
```
This does remove the check for blank lines, but it appears `GetCostBreakdown` never returns any blank lines.
It also removes spacing between the breakdown entries:
<img width="359" height="290" alt="image" src="https://github.com/user-attachments/assets/41058822-7a08-4f5a-b281-0df3c5ac4d5b" />

Reply:
Good suggestion. I forgot about the WrappedText :)